src/Content/Text/BBCode.php

```diff
@@ -1067,6 +1067,43 @@ function ($match) use ($callback, $uriid) {
 		return $return;
 	}
 
+	/**
+	 * Convert complex IMG and ZMG elements
+	 *
+	 * @param [type] $text
+	 * @param integer $simplehtml
+	 * @param integer $uriid
+	 * @return string
+	 */
+	private static function convertImages(string $text, int $simplehtml, int $uriid = 0):string
+	{
+		DI::profiler()->startRecording('rendering');
+		$return = preg_replace_callback(
+			"/\[[zi]mg(.*?)\]([^\[\]]*)\[\/[zi]mg\]/ism",
+			function ($match) use ($simplehtml, $uriid) {
+				$attribute_string = $match[1];
+				$attributes = [];
+				foreach (['alt', 'width', 'height'] as $field) {
+					preg_match("/$field=(['\"])(.+?)\\1/ism", $attribute_string, $matches);
```

Comment:
This covers `field="value"` and `field='value'`, but do we know if `field=value` exists as well?

Reply:
I currently don't find the Zap repository. But yesterday I had a quick look into it. AFAIK there are always quotes.